src/modules/fstab/main.py

```diff
@@ -266,11 +268,11 @@ def generate_fstab_line_info(self, partition):
 
     def print_fstab_line(self, dct, file=None):
         """ Prints line to '/etc/fstab' file. """
-        line = "{:41} {:<14} {:<7} {:<10} 0       {}".format(dct["device"],
-                                                             dct["mount_point"],
-                                                             dct["fs"],
-                                                             dct["options"],
-                                                             dct["check"],
+        line = "{:41} {:<14} {:<7} {:<10} 0      {}".format(dct["device"],
```

Comment:
You're moving these by one space just to satisfy the style checker -- why not compact the fstab line all the way to `"{:41} {:<14} {:<7} {:<10} 0 {}"` and leave those extra spaces out?